pkg/reconciler/trigger/trigger.go

```diff
@@ -221,7 +221,7 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, t *eventingv1.Trigger) p
 		var dlxName *string
 		if t.Spec.Delivery != nil && t.Spec.Delivery.DeadLetterSink != nil {
 			dlxName = ptr.String(naming.TriggerDLXExchangeName(t))
-		} else {
+		} else if broker.Spec.Delivery != nil && broker.Spec.Delivery.DeadLetterSink != nil {
 			dlxName = ptr.String(naming.BrokerExchangeName(broker, true))
 		}
```

Comment:
Nitpick, line 222 is the same as line 149, could be initialize dlxName before the whole block of creating trigger dlx and dlq, to avoid repeating this if statement?

Reply:
yep, thanks!